### ISSUE TITLE
Downloaded dist version of atk from website produces error

### ISSUE THREAD

**Issue by bedengler** [atk4/ui#838]
I'm just diving into atk. 2 days ago I downloaded it from https://www.agiletoolkit.org/, loaded it up and was able to see the admin interface.

After running a composer update on the directory, composer downgraded atk/data to version 1.3.7.

So I went again to the website today, downloaded https://www.agiletoolkit.org/dist/agiletoolkit-1.7.1.tgz, loaded admin.php and got this error:
Fatal Error
atk4\ui\Exception: Table does not have any columns defined

See also attached screenshot.
![Bildschirmfoto 2019-11-12 um 16 51 14](https://user-images.githubusercontent.com/17435772/68686922-ade08200-056c-11ea-9761-8b0956b861c4.jpg)

I've tried opening it in different browsers, deleting cookies etc.
I then recovered the old version from my Timemachine backup and everything worked again.
Am I doing something wrong or is the deployed version damaged?

**Comment by acicovic** [comment]
Hi,
There shouldn't be any problem in the file. I checked it and it seems to be valid. I'm not sure why you get this error but it could sometimes come by other invalid markup in code. Also, if you download things with composer and then manually overwrite, this could also cause you problems with incompatible component versions so be careful when you do this.

**Comment by bedengler** [comment]
Thanks for your reply.
I did it the other way round: 
- First downloaded original, worked.
- Then did a composer update which downgraded atk/data. 
- Then I deleted the whole directory, downloaded the file again (which was obviously a new version).
- Then I got the error from above. No manual changes in code so far. I just called the admin.php of the original atk files.
- Then I rolled back, means: I deleted the whole directory again, took the one from the backup (which was unchanged as well) and everything worked again. 
So i assumed there has to be a problem in the (new) file from the atk server...
